pytorch_lightning/plugins/training_type/horovod.py

```diff
@@ -90,12 +91,17 @@ def _filter_named_parameters(model, optimizer):
             return [(name, p) for name, p in model.named_parameters() if p in opt_params]
 
         # Horovod: wrap optimizers to perform gradient aggregation via allreduce
-        optimizers = [
-            hvd.DistributedOptimizer(
-                optimizer, named_parameters=_filter_named_parameters(self.lightning_module, optimizer)
-            ) for optimizer in optimizers
-        ]
-        self.lightning_module.trainer.accelerator.optimizers = optimizers
+        h_optimizers = []
+        for optimizer in optimizers:
+            if 'horovod' not in str(optimizer.__class__):
+                h_optimizers.append(
+                    hvd.DistributedOptimizer(
+                        optimizer, named_parameters=_filter_named_parameters(self.lightning_module, optimizer)
+                    )
+                )
+            else:
+                h_optimizers.append(optimizer)
+        self.lightning_module.trainer.accelerator.optimizers = h_optimizers
```

Comment:
@awaelchli this could be another reason why we should merge accelerators and training type plugins

Reply:
merge in what sense. do you mean make it a property of the plugin, so the call would change to 
`self.accelerator.optimizers = ...`, right?